source/content/solr-drupal-9.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Enabling Solr 8 on Drupal 9
+description: Detailed information on using Pantheon Search with Solr 8 on Drupal 9
+cms: "Drupal 9"
+categories: [integrate]
+tags: [solr, search, modules]
+contributors: [carolynshannon]
+reviewed: "2021-06-17"
+---
+[Apache Solr](/solr) is a system for indexing and searching site content.
+
+<Alert title="Important Note" type="info">
+
+**Pantheon Search with Solr 8** derives from Solr and can perform full-text content searching in a single language.
+
+<Partial file="solr-version.md" />
+
+If your search needs include geospatial search, emojis, or multilingual search, consider [OpenSolr](/opensolr) or another alternative search.
+
+Pantheon Search with Solr 8 supports [Search API Solr x.x.x](https://www.drupal.org/project/search_api_solr), and requitres the companion Search API Pantheon module, configiured following the installation directions below.
+
+</Alert>
+
+## Before You Begin
+
+Be sure that you:
+
+- Enable Solr in the Pantheon Site Dashboard: **Settings** > **Add Ons** > **Apache Solr Index Server: Add**.
+
+- Install [Composer](https://getcomposer.org/)
+
+- Create a Drupal 9 with Integrated Composer managed site on Pantheon following the [Drupal 9](/drupal-9) guide, or migrate to Drupal 9 with Integrated Composer using the (Drupal 9 Migration Guide)[/guides/drupal-9-migration].
```

Comment:
> Create a Drupal 9 **site** with ...

Reply:
Having a D9 site w Integrated Composer is a requirement for using this doc, and has been specified in the "Will this guide work for your site?" section.